defensics/tooling.py

```diff
@@ -0,0 +1,181 @@
+# TODO: add support for external tools: btmon, serial port output,
+#       file management with result sorting
+import subprocess
+import threading
+import serial
+import os
+import time
+import logging
+import signal
+from queue import Queue
+
+pwd = ''
```

Comment:
Overall the idea sounds good but lets use an env variable to pass the password to script.

Reply:
Or use a configuration file.

Reply:
As discussed offline, configuration file will be used